src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { User } from '../../type/User';
+import './UserInfo.scss';
+
+type Props = {
+  completed: boolean;
+  user: User | null
```

Comment:
let's make it reuqired prop
```suggestion
  user: User;
```